ansible/roles/ocp-workload-starter-workshop/tasks/per_user.yml

```diff
@@ -83,16 +83,20 @@
       tags: gogs
 
     - name: push catalog to git repository in Gogs for user "{{ my_user }}"
-      shell: |
-        git remote add {{ my_user }} http://{{ my_user }}:{{ user_gogs_password }}@{{ gogs_hostname }}/{{ my_user }}/{{ reponame_local }}.git
+      shell: >-
```

Comment:
Why was this changed from literal to folded? It was not good before to be in a single `shell` task, but this will break it as is.

Reply:
Line length constraints bc of the add url. Should not break .. double line in `>` converts to single new line. The yaml to json conversion looks the same. Could also separate the commands using `;` or `&&` as we would in shell.

Reply:
but if you use folded, it will just string all of these lines together with a whitespace between them. These are all separate commands and, as you noted, this can be fixed with an `&&` option, but as of right now this would be broken.

Reply:
Double newline in folded = single newline in output. Single newline in `|` operator is the same as double newline in `>-` operator. That's why single newlines were converted to double newlines when the operator was changed from '|' to '>-`. It should not break.  The yaml to json conversion also looks the same.

Reply:
Oh, you are right!